dev_setup.py

```diff
@@ -15,6 +15,7 @@
 build_dir = source_dir / "build"
 
 print("=== System packages (sudo apt install ...) ===")
+check_call(["sudo", "apt", "update"])
```

Comment:
Good add! Maybe move this to just before the call to sudo apt install (line 26), so sudo isn't invoked if the packages are all good?

Reply:
changed